### PR TITLE
Fix error pipeline 11-08 and change version of webdriver

### PR DESCRIPTION
**Pull request by delfinoroNTTDATA**
Fix error pipeline (test 99) and change versions of web driver in pom

### CODE REVIEW

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/DownloadFileMittentePagoPATest.java [hunk 1]
```diff
@@ -158,7 +158,12 @@ public void siSelezionanoIlFileDaScaricare(String nomeFile) {
         this.datiNotifica = dataPopulation.readDataPopulation("datiNotifica.yaml");
 
         DettaglioNotificaSection dettaglioNotificaSection = new DettaglioNotificaSection(this.driver);
-        dettaglioNotificaSection.clickLinkAttestazioneOpponibile(nomeFile);
+
+        if(nomeFile.equals("Attestazione opponibile a terzi: avvenuto accesso")){
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/DownloadFileMittentePagoPATest.java]: insert string in a variable

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java [hunk 2]
```diff
@@ -257,8 +257,56 @@ public void nellaSectionAllegatiCliccareSulBottoneInvia() {
...
+            aggiornamentoNumeroProtocolloAllegati();
+            logger.error("Si vede il messaggio di dati non corretti");
+            Assert.fail("Si vede il messaggio di dati non corretti");
+        }
     }
+    private void aggiornamentoNumeroProtocolloAllegati() {
+        logger.info("Aggiornamento del numero protocollo");
+
+        DataPopulation dataPopulation = new DataPopulation();
+        String numeroProtocolOld = dataPopulation.readDataPopulation("datiNotifica.yaml").get("numeroProtocollo").toString();
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java]: insert string in a variable, path and 'numeroProtocollo'

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/DownloadFileMittentePagoPATest.java (hunk 1 above)
Comment (delfinoroNTTDATA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/DownloadFileMittentePagoPATest.java]: done

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java (hunk 2 above)
Comment (delfinoroNTTDATA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java]: done

**Review by leleOFA — APPROVED** [review]

**Pull request merged**